scripts/verify_build.sh

```diff
@@ -63,7 +63,8 @@ done
 if [[ "$build_commands" != "" ]]; then
     # We override HAB_CACHE_KEY_PATH to ensure we only see the key we
     # generated in this build
-    HAB_ORIGIN=chef HAB_CACHE_KEY_PATH=$RESOLVED_RESULTS_DIR DO_CHECK=true hab studio run "source .studiorc; set -e; $build_commands"
+    export HAB_DOCKER_OPTS="--label buildkitejob=$BUILDKITE_JOB_ID"
```

Comment:
I can't tell if we want `--tty` or not anymore.

Reply:
probably leave it
change 1 thing at a time